### DISCUSSION TITLE
How to vizualize both Properties and Data of a table simultaneously

### DISCUSSION THREAD

**Discussion by TelKel81** [dbeaver/dbeaver#20739] [Q&A]
When viewing a Table, it seems like Properties and Data are Tabs in a TabPane, and therefore are not windows that be placed side by side, or docked, etc.

Am I missing something ?

**Comment by LonwoLonwo** [comment]
Hello @TelKel81 

What kind of data are you looking for in the Properties tab?

Maybe using [Metadata panel](https://github.com/dbeaver/dbeaver/wiki/Panels#metadata-panel) will help?

![2023-07-30 22_27_42-DBeaver Ultimate 23 1 3 - Customer](https://github.com/dbeaver/dbeaver/assets/45152336/e7594ba8-9e6c-484d-b5da-c17de7be58d9)
